mathics/builtin/quantities.py

```diff
@@ -185,88 +175,63 @@ class QuantityMagnitude(Builtin):
 
     >> QuantityMagnitude[Quantity[{10,20}, "Meters"]]
      = {10, 20}
-
-    #> QuantityMagnitude[Quantity[1, "meter"], "centimeter"]
-     = 100
-
-    #> QuantityMagnitude[Quantity[{3,1}, "meter"], "centimeter"]
-     = {300, 100}
-
-    #> QuantityMagnitude[Quantity[{300,100}, "centimeter"], "meter"]
-     = {3, 1}
-
-    #> QuantityMagnitude[Quantity[{3, 1}, "meter"], "inch"]
-     = {118.11, 39.3701}
-
-    #> QuantityMagnitude[Quantity[{3, 1}, "meter"], Quantity[3, "centimeter"]]
-     = {300, 100}
-
-    #> QuantityMagnitude[Quantity[3,"mater"]]
-     : Unable to interpret unit specification mater.
-     = QuantityMagnitude[Quantity[3,mater]]
     """
 
     summary_text = "get magnitude associated with a quantity."
 
-    def eval(self, expr, evaluation: Evaluation):
-        "QuantityMagnitude[expr_]"
+    def eval_List1(self, expr, evaluation: Evaluation):
```

Comment:
This and `eval_list2 ()` and `validate_unit` should probably get moved to the `mathics.eval.quantities` module.